Firestore/core/src/remote/stream.cc

```diff
@@ -295,7 +295,11 @@ void Stream::HandleErrorStatus(const Status& status) {
 void Stream::OnStreamFinish(const Status& status) {
   EnsureOnQueue();
 
-  LOG_DEBUG("%s Stream error: '%s'", GetDebugDescription(), status.ToString());
+  if(!status.ok()) {
```

Comment:
Nit: there should be a space between `if` and the opening brace, you probably need to run `style.sh`.

Reply:
Done.